jellyfin_kodi/helper/playutils.py

```diff
@@ -399,6 +399,12 @@ def get_transcoding_video_codec(self):
         if settings("transcode_vc1.bool"):
             codecs.remove("vc1")
 
+        if settings("transcode_av1.bool"):
+            codecs.remove("av1")
+        else:
+            if settings("videoPreferredCodec") == "AV1":
```

Comment:
This should be an `elif`.
Would be cleaner to implement this the same way as HEVC is now done.

I am undecided as how to handle the race condition of setting preferred codec while asking to transcode that same codec.

Reply:
I think the best solution would be hiding the transcode option from the menu and assuming false when it's preferred.

Reply:
That is probably the best option, yea